### PR TITLE
unzip fix bug

### PR DESCRIPTION
**Pull request by esharf**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/21171
fixes: https://github.com/demisto/etc/issues/21258

## Minimum version of Demisto
- [x] 4.5.0

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Tests

### CODE REVIEW

**Comment by bakatzir** [comment]
@kirbles19 
CHANGELOG:
https://github.com/demisto/content/blob/22e6fca8db9656eaacf238864559b695273cefa0/Scripts/UnzipFile/CHANGELOG.md

File: Scripts/UnzipFile/UnzipFile_test.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+data_test_extract = [
+    ('testZip.yml', None),
+    ('ScanSummary.txt', None),
+    ('item.png', None),
+    ('fix_unzip.png', 'demisto'),
+]
+
+
+@pytest.mark.parametrize('file_name, password', data_test_extract)
+def test_extract(file_name, password):
```
Comment (anara123) [inline Scripts/UnzipFile/UnzipFile_test.py]: the test is a bit hard to read.
explain in steps what are you testing
use the template:
```
Given
- valid zip file
- empty folder _dir

When
- run extract on that zip file and export the internal content to _dir

Then
- ensure zip file content have be saved at _dir directory with the original filename
- ensure that the saved file has expected content
```

File: TestPlaybooks/playbook-UnzipFile_-_Test.yml [hunk 2]
```diff
@@ -0,0 +1,245 @@
+id: UnzipFile - Test
+version: 10
```
Comment (anara123) [inline TestPlaybooks/playbook-UnzipFile_-_Test.yml]:
```suggestion
version: -1
```

**Comment by kirbles19** [comment]
@esharf  Done.

**Comment by anara123** [comment]
@esharf your build fails 
```
Number of failed tests - 1:
	 - UnzipFile - Test (Mock Disabled)
```
maybe you should change the id of your test playbook and remove whitespaces

**Pull request merged**